### PR TITLE
Increase exec timeout

### PR DESCRIPTION
**Pull request by ajlanghorn**
The default value of the timeout parameter used in the exec type in Puppet is
300secs. freshclam takes longer than 300secs to complete at times. If this
happens when a box is brought up, the provisioning may fail. Even though
freshclam will re-run after a while once the box has booted, it is far better to
ensure that we get a successful output from freshclam during provision, rather
than an error relating to it timing out.

This pull request increases the timeout value from 300s to 600s.

### CODE REVIEW

File: manifests/run_freshclam.pp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #
 class clamav::run_freshclam {
   exec { '/usr/bin/freshclam --quiet':
+    timeout     => '600',
```
Comment (dcarley) [inline manifests/run_freshclam.pp]: This is an integer so it should be unquoted.

**Comment by dcarley** [comment]
Can you tell how long `freshclam` has been taking? Did raising this prevent it ever happening again?

**Comment by ajlanghorn** [comment]
@dcarley Yes - it works fine after I increased the timeout to 600, so it's taking somewhere between 300 and 600. I could play further to reduce the 600 to a more realistic number, if you like? From looking at the freshclam.log, it looks like it takes about 6mins or so, so I could change to about that?

**Comment by dcarley** [comment]
We should choose a non-arbitrary number based on some evidence. If we don't think it should take 10mins then we needn't increase it to that ;)

**Comment by ajlanghorn** [comment]
@dcarley All done. Tests out fine, too.

**Pull request merged**